src/options/options.html

```diff
@@ -154,6 +154,19 @@ <h2 data-i18n="__MSG_subtitleEntryPreview__">Entry design</h2>
 				</ul>
 			</section>
 
+			<section>
+				<h1>Notifications</h1>
+
+				<ul>
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="send" data-optiongroup="notifications" name="send">
+							<label for="send">Display Desktop notifications</label>
```

Comment:
```suggestion
							<label for="send">Display notifications on extension installation and upgrade.</label>
```

The user is otherwise questioned with _what_ is shown in these notifications or _whats_  even triggers them.

Also would not say desktop, that is an irrelevant implementation detail and may not even be true in case you manage to run this code on Android ;P

Reply:
As I mentioned above, this is for all notifications, including both these "install and update" and the Thunderbird specific notification added in #72.

Note that the installation notification would not actually be possible to disable, as it would have of course already been shown by the time the user opened the options page. The upgrade notifications should be extremely infrequent, so I doubt users would want to disable them. However, users may want to disable any tip notifications added as part of https://github.com/TinyWebEx/RandomTips/issues/9, which is why I added this option.

Reply:
> and the Thunderbird specific notification added in https://github.com/rugk/unicodify/pull/72.

This is an error shown based on a user-initiated action, as such IMHO, it should _not be able to be disabled_, i.e. not controlled by that option, but always be shown.

> Note that the installation notification would not actually be possible to disable, as it would have of course already been shown by the time the user opened the options page

Of course, fair point hehe… yeah, then we can ignore that.

_Though_ technically, it would be possible to implement it like that and I would also do so, because there _may_ be an edge-case where that is not to be shown: When you have installed and configured the extension _and_ you connect a new Firefox to Firefox Sync, and your sync runs, it may happen that the extension settings/data is already syncronized, before the extension is installed? Though I don't know how Firefox handles that and in what order stuff is syncronized, so well… it also does not matter. Just keep it simple, you know.

Reply:
> This is an error shown based on a user-initiated action, as such IMHO, it should _not be able to be disabled_

OK, that is fine with me and it would be an easy change.

> _Though_ technically, it would be possible to implement it like that and I would also do so, because there _may_ be an edge-case where that is not to be shown

Good point. It actually is already implemented like this, so your scenario if the sync ran before the new install in the installation notification would not show. Is there any other specific change you would like me to make here?